components/LoadingWrapper/LoadingWrapper.tsx

```diff
@@ -3,14 +3,16 @@ import { Loading } from '../../.storybook/stories/Loading';
 
 type LoadingWrapperProps = {
   isLoading: boolean;
+  text?: string;
 };
 
 export const LoadingWrapper: FC<PropsWithChildren<LoadingWrapperProps>> = ({
   isLoading,
+  text = '',
```

Comment:
If you add a default value here, it can't be undefined in the `Loading` component